docs/website/docs/dlt-ecosystem/destinations/snowflake.md

```diff
@@ -310,6 +310,28 @@ Above we set `csv` file without header, with **|** as a separator and we request
 You'll need those setting when [importing external files](../../general-usage/resource.md#import-external-files)
 :::
 
+### Query Tagging
+`dlt` tags sessions that execute loading jobs with following job properties:
+* **source** - name of the source (identical with the name of `dlt` schema)
+* **resource** - name of the resource
+* **table** - name of the table loaded by the job
+* **load_id** - load id of the job
```

Comment:
While we're at it, should we also add pipeline name and dataset name to the tags?

Reply:
yes this is possible